Mathlib/GroupTheory/Perm/Support.lean

```diff
@@ -326,6 +333,46 @@ theorem support_inv (σ : Perm α) : support σ⁻¹ = σ.support := by
 theorem apply_mem_support {x : α} : f x ∈ f.support ↔ x ∈ f.support := by
   rw [mem_support, mem_support, Ne, Ne, apply_eq_iff_eq]
 
+/-- The support of a permutation is invariant -/
+theorem isInvariant_of_support_le {c : Perm α} {s : Finset α}
+    (hcs : c.support ≤ s) (x : α) : x ∈ s ↔ c x ∈ s := by
```

Comment:
Ditto

Reply:
In fact, maybe not: the issue, that appears later, is that the argument to`Equiv.Perm.subtypePerm` is written in this direction.